app/com/linkedin/drelephant/tuning/FitnessComputeUtil.java

```diff
@@ -48,20 +49,38 @@
 /**
  * This class computes the fitness of the suggested parameters after the execution is complete. This uses
  * Dr Elephant's DB to compute the fitness.
- * Fitness is : Resource Usage/Input Size in GB
+ * Fitness is : Resource Usage/(Input Size in GB)
  * In case there is failure or resource usage/execution time goes beyond configured limit, fitness is computed by
  * adding a penalty.
  */
 public class FitnessComputeUtil {
   private static final Logger logger = Logger.getLogger(FitnessComputeUtil.class);
   private static final String FITNESS_COMPUTE_WAIT_INTERVAL = "fitness.compute.wait_interval.ms";
+  private static final String IGNORE_EXECUTION_WAIT_INTERVAL = "ignore.execution.wait.interval.ms";
   private static final int MAX_TUNING_EXECUTIONS = 39;
   private static final int MIN_TUNING_EXECUTIONS = 18;
-  private Long waitInterval;
+  private Long fitnessComputeWaitInterval;
+  private Long ignoreExecutionWaitInterval;
 
   public FitnessComputeUtil() {
     Configuration configuration = ElephantContext.instance().getAutoTuningConf();
-    waitInterval = Utils.getNonNegativeLong(configuration, FITNESS_COMPUTE_WAIT_INTERVAL, 5 * AutoTuner.ONE_MIN);
+    fitnessComputeWaitInterval =
+        Utils.getNonNegativeLong(configuration, FITNESS_COMPUTE_WAIT_INTERVAL, 5 * AutoTuner.ONE_MIN);
+    ignoreExecutionWaitInterval =
```

Comment:
Can you add a comment on these constants?

Reply:
Done